lib/mpu9250/MPU9250.cpp

```diff
@@ -110,16 +120,12 @@ void MPU9250::getMagnetometerAdjustments(float adjustments[3]) {
     adjustments[2] = asaz;
 }
 
-uint8_t MPU9250::getAddr() {
-    return devAddr;
-}
-
 /** Verify the I2C connection.
  * Make sure the device is connected and responds as expected.
  * @return True if connection is valid, false otherwise
  */
 bool MPU9250::testConnection() {
-    return getDeviceID() > 0;
+    return getDeviceID() == 0x71;
```

Comment:
There's also MPU-9255 that has 0x73 device ID.
https://stanford.edu/class/ee267/misc/MPU-9255-Register-Map.pdf

Reply:
Thank you!